FCCee/IDEA/compact/IDEA_o1_v03/IDEA_o1_v03.xml

```diff
@@ -42,7 +42,7 @@
   <include ref="Vertex_IDEA_o1_v01.xml"/>
 
   <!-- Import drift chamber -->
-  <include ref="DriftChamber_o1_v02.xml"/>
+  <!-- <include ref="DriftChamber_o1_v02.xml"/> -->
```

Comment:
why is this commented out?

Reply:
Oh I forgot to place it back. I'll update it.

I commented it out to test running DRC simulation.
I did it because when compiling the k4geo I got this warning message (even without any new code, freshly cloned from k4geo main branch) 

`CMake Warning at CMakeLists.txt:106 (message):
   Subdetector DriftChamber_o1_v02.cpp will not be built because header file
   was not found`

and if I don't comment drift chamber out and start simulation (even w/o implementation of DRC) I got this error

```
****************************************************
*  A runtime error has occured :
*    Compact: Failed to execute subdetector creation plugin. 		No factory with name Create(DriftChamber_o1_v02_T) for type DriftChamber_o1_v02_T found.
		Please check library load path and/or plugin factory name.
*  the program will have to be terminated - sorry.
****************************************************
```

So I just commented out drift chamber and ran the ddsim with DRC.

Reply:
Hi Sungwon, yes this is because the drift chamber builder is not yet included in the stable stack (last one is from March), we currently run this detector with the nightlies (`source /cvmfs/sw-nightlies.hsf.org/key4hep/setup.sh`)